modules/tinymce/src/core/main/ts/api/WindowManager.ts

```diff
@@ -144,15 +144,18 @@ const WindowManager = (editor: Editor): WindowManager => {
      * Opens a new window.
      *
      * @method open
-     * @param {Object} args For a list of options, see: <a href="https://www.tiny.cloud/docs/tinymce/6/dialog-configuration/#configurationoptions">Dialog - Configuration options</a>.
+     * @param {Object} config For information on the available options, see: <a href="https://www.tiny.cloud/docs/tinymce/6/dialog-configuration/#configurationoptions">Dialog - Configuration options</a>.
+     * @param {Object} params (Optional) For information on the available options, see: <a href="https://www.tiny.cloud/docs/tinymce/6/dialog-configuration/#configurationparameters">Dialog - Configuration parameters</a>.
```

Comment:
@brian-forte At the moment, I am pointing to `https://www.tiny.cloud/docs/tinymce/6/dialog-configuration/#configurationparameters` for the param options documentation which doesn't exist yet. Just wondering how you were envisioning documenting the param options and if this will have to be a different URL

Reply:
> @brian-forte At the moment, I am pointing to `https://www.tiny.cloud/docs/tinymce/6/dialog-configuration/#configurationparameters` for the param options documentation which doesn't exist yet. Just wondering how you were envisioning documenting the param options and if this will have to be a different URL

### context

I have three tickets open for documenting these new parameters:

1. [DOC-2099](https://ephocks.atlassian.net/browse/DOC-2099)
	* Document how to configure a custom dialog to not render its footer
3. [DOC-2101](https://ephocks.atlassian.net/browse/DOC-2101)
	* Document new `persistent` option in `WindowParams`
5. [DOC-2102](https://ephocks.atlassian.net/browse/DOC-2102)
	* Document new `streamContent` property for the `iframe` dialog component which controls dialog `setData()` behavior.

[DOC-2099](https://ephocks.atlassian.net/browse/DOC-2099) has a PR up: [PR2099](https://github.com/tinymce/tinymce-docs/pull/2807).

And, from that first PR, you can see my initial approach was a flat structure: just add the option documentation to the existing docs. (This especially when you note I’ve already got a new [Dialog position](https://tiny.cloud/docs/tinymce/6/dialog-configuration/#dialog-position) addition in place and published.)

### all this said

Putting `persistent` and `streamContent` into a _Configuration parameters_ sub-section seems like a sensible approach.

The emergent page structure would be:

* Configuring custom dialogs
  - Basic example
  - Options
    * Event callback functions
  - Configuration parameters
    * `persistent`
    * `streamContent`
    * `button` (as setting for turning the dialog footer on or off)
  - Dialog position
    * `inline`
    * Basic example with the inline argument added

### URL note

The actual URL that will roll out of all this is `http://www.tiny.cloud/docs/tinymce/6/dialog-configuration#configuration-parameters` (note the hyphen between _configuration_ and _parameters_).